### DISCUSSION TITLE
Currency codes don't really belong under Accounting module

### DISCUSSION THREAD

**Discussion by rivettp** [edmcouncil/fibo#1394] [Foundations (FND)]
I wonder whether [Currency codes](https://spec.edmcouncil.org/fibo/ontology/FND/Accounting/ISO4217-CurrencyCodes/) really belong under [Accounting](https://spec.edmcouncil.org/fibo/ontology/FND/Accounting/MetadataFNDAccounting/AccountingModule) - they are far more general in scope.

_Originally posted by @rivettp in https://github.com/_render_node/MDIzOlB1bGxSZXF1ZXN0UmV2aWV3VGhyZWFkMjM1MzgyNjEzOnYy/pull_request_review_threads/discussion_

**Comment by ElisaKendall** [comment]
The individual currency codes depend heavily on the CurrencyAmount ontology - would you have those classes and properties moved as well?  they are used throughout FIBO and by a number of FIBO users.  So you think that they should be in a separate standard outside of FIBO, managed like LCC by the OMG, but not a part of FIBO?  Otherwise I see no point in unwinding them from the CurrencyAmount ontology.

**Comment by ElisaKendall** [comment]
The issue is really the name of this module, which should more properly be Finance, not Accounting, based on its contents.  But, changing the module name would impact URIs across FIBO, which may not be worth doing at this point.
